src/index.js

```diff
@@ -33,8 +26,39 @@ items.forEach((item) => {
 
   input.addEventListener('change', () => {
     updateCompleted(item, input);
-    saveLocalStorage(items);
+    saveLocalStorage(itemsArray);
   });
 
   todos.appendChild(div);
+  const vertical = div.querySelector('.vertical');
+  const trash = div.querySelector('.trash');
+  const edit = div.querySelector('.edit');
+  edit.addEventListener('change', () => {
+    editTodo(itemsArray, div.id, edit.value);
+    saveLocalStorage(itemsArray);
+  });
+  vertical.addEventListener('click', () => {
+    vertical.style.display = 'none';
+    trash.style.display = 'block';
+  });
+  trash.addEventListener('click', () => {
+    div.remove();
+    itemsArray = itemsArray.filter((i) => removeTodos(i, div.id));
+    saveLocalStorage(itemsArray);
+  });
+});
+
+const form = document.getElementById('form-id');
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const { value } = form.querySelector('input');
+  addTodo(itemsArray, value);
+  saveLocalStorage(itemsArray);
+  window.location.reload();
+});
+
+const clearButton = document.querySelector('#button');
```

Comment:
- [ ] And after fixing the first issue with `form-id` this querySelector throws a similar error in the console because there are no elements with the id `button` :pushpin:
![image](https://user-images.githubusercontent.com/81522435/145215767-97a93ef3-9abe-4396-ac87-7705d22b9e9c.png)
I tried fixing this issue by adding the `button` id to the `<button>` element :smiley: